tools/val/val.cpp

```diff
@@ -44,6 +44,7 @@ NOTE: The validator is a work in progress.
   --max-function-args              <maximum number arguments allowed per function>
   --max-control-flow-nesting-depth <maximum Control Flow nesting depth allowed>
   --max-access-chain-indexes       <maximum number of indexes allowed to use for Access Chain instructions>
+  --relax-struct-store             Allow store from one struct type to a different type with compatible layout and members.
```

Comment:
Avoid going past 80 columns.

Reply:
I fixed the description for the option I added.  I can fix the other options too, but you'll have to tell me what you expect.

Reply:
Thanks.  Just keep the fixes to the one you added.